### PR TITLE
Swift 2.3

### PR DESCRIPTION
**Pull request by sendhil**
This is taking over for https://github.com/wordpress-mobile/WordPress-iOS/pull/5977. The problem with that PR and branch is that it had develop merged in by accident as the swift 2.3 branch was always targeting the release branch so we could ship it right away.

cc @aerych, @koke @jleandroperez, @diegoreymendez 

@aerych and @koke - I pushed the changes you asked for in the previous thread.

@diegoreymendez - could you jump in and update the Aztec code to work correctly? The code is still just commented out so the build would work but if you could update it to work that'd be awesome.

### CODE REVIEW

**Comment by kurzee** [comment]
@diegoreymendez also seeing via `pod install`:

```
[!] Error installing WordPress-Aztec-iOS
[!] /usr/bin/git checkout --quiet f40ba3003b6d8949019defb407a8e68db3a91e53

fatal: reference is not a tree: f40ba3003b6d8949019defb407a8e68db3a91e53
```

**Comment by jleandroperez** [comment]
@kurzee @diegoreymendez i've updated the Aztec mapping to the the latest commit within the .3 branch.

Please, give it a shot whenever you can!

**Comment by diegoreymendez** [comment]
> could you jump in and update the Aztec code to work correctly? The code is still just commented out so the build would work but if you could update it to work that'd be awesome.

I'll take care of that.

Just one note for @koke, though:

Aztec is an alpha feature that's far from being ready, and that was added due to the necessity to showcase it at the GM if possible.  It doesn't make sense to add feedback on its code for the time being, as most of what's in there had to be rushed in, and the fixes we're applying now respond to the poor timing of the Xcode 8 update.

My recommendation for anyone reviewing the code: let's try to keep the feedback down to what really needs to be changed for things to work again.  Then we can worry about style.

**Comment by diegoreymendez** [comment]
I updated the shared pod to v0.6.1 since Aztec needed that, and also updated the whole Aztec VC with the latest from `develop`.

File: WordPress/Classes/ViewRelated/Views/SeparatorsView.swift [hunk 1]
```diff
@@ -86,41 +86,41 @@ public class SeparatorsView : UIView
 
         let scale = UIScreen.mainScreen().scale
         let ctx = UIGraphicsGetCurrentContext()
-        CGContextClearRect(ctx, rect)
-        CGContextSetShouldAntialias(ctx, false)
+        CGContextClearRect(ctx!, rect)
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Views/SeparatorsView.swift]: It would be cleaner to add the `!` to `UIGraphicsGetCurrentContext()`

**Pull request merged**